Dockerfile.test.rhel7

```diff
@@ -0,0 +1,63 @@
+FROM registry.access.redhat.com/ubi7/ubi
```

Comment:
Can you please add a comment at the top of the file explaining what this file does and is trying to accomplish (configure dbus for example). It has a look of good and complex stuff in here